package/main.yml

```diff
@@ -31,15 +31,6 @@ parameters:
       version: v1.2.0
       path: component
 
-  syn_openshift_distributed_tracing:
```

Comment:
In theory, you might have been able to do it with some creative reclass references , but your approach is cleaner & more extensible. Sketch for reclass-based approach (untested, not sure if openshift4-operators would complain about an empty instance config):

```yaml
  _syn_openshift_distributed_tracing:
    "True":
      useCustomNamespace: true
      subscription:
        name: jaeger-product
        channel: ${pkg.openshift4_service_mesh:distributed_tracing:channel}
        spec:
          source: redhat-operators
          installPlanApproval: ${pkg.openshift4_service_mesh:distributed_tracing:installPlanApproval}
    "False": {}

  syn_openshift_distributed_tracing: ${_syn_openshift_distributed_tracing:${pkg.openshift4_service_mesh.distributed_tracing.enabled}}

```